extensions/awt/deployment/src/main/java/io/quarkus/awt/deployment/AwtProcessor.java

```diff
@@ -258,18 +252,11 @@ JniRuntimeAccessBuildItem setupJava2DClasses(NativeImageRunnerBuildItem nativeIm
         // A new way of dynamically loading shared objects instead
         // of baking in static libs: https://github.com/oracle/graal/issues/4921
         classes.add("sun.awt.X11FontManager");
-        if (v.javaVersion.feature() != 19) {
-            classes.add("java.awt.GraphicsEnvironment");
-            classes.add("sun.awt.X11GraphicsConfig");
-            classes.add("sun.awt.X11GraphicsDevice");
-            classes.add("sun.java2d.SunGraphicsEnvironment");
-            classes.add("sun.java2d.xr.XRSurfaceData");
-        }
-
-        // Added for JDK 19+ due to: https://github.com/openjdk/jdk20/commit/9bc023220 calling FontUtilities
-        if (v.jdkVersionGreaterOrEqualTo("19")) {
-            classes.add("sun.font.FontUtilities");
```

Comment:
No, that's moved to the new AwtFeature (https://github.com/quarkusio/quarkus/pull/43829/files#diff-086ee2e4b2043822961920521dd5be21c10c6805629f87f5b79d5836eb978e28R14).

The idea is to avoid version checks at Quarkus deployment phase (were we are still not sure what GraalVM version will be used).